packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -230,11 +230,14 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 		batch = false,
 		metadata?: any,
 		compression?: string,
+		referenceSequenceNumber?: number,
```

Comment:
Can we add a submitBatch function that Container calls instead of this? (Maybe in a separate PR)